packages/swap/prisma/migrations/20250116152326_backfill_channel_data_to_swap_request/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- backfill simple columns from deposit channel to swap request
```

Comment:
we want to merge this once 1.8 is released to mainnet to backfill the new properties of the `SwapRequest` for existing swaps